FlorenceBML/BMLor403/BMLor403.xml

```diff
@@ -34,8 +35,421 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                            <locus from="137" to="144"/> 
+                                        </item>
+                                        <item xml:id="q21" n="21">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="145" to="152"/> 
+                                        </item>
+                                        <item xml:id="q22" n="22">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="153" to="160"/> 
+                                        </item>
+                                        <item xml:id="q23" n="23">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="161" to="168"/> 
+                                        </item>
+                                        <item xml:id="q24" n="24">
+                                            <dim unit="leaf">6</dim>
+                                            <locus from="169" to="174"/>
+                                        </item>
+                                        <item xml:id="q25" n="25">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="175" to="182"/> 
+                                        </item>
+                                        <item xml:id="q26" n="26">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="183" to="190"/> 
+                                        </item>
+                                        <item xml:id="q27" n="27">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="191" to="198"/> 
+                                        </item>
+                                        <item xml:id="q28" n="28">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="199" to="206"/> 
+                                        </item>
+                                        <item xml:id="q29" n="29">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="207" to="214"/> 
+                                        </item>
+                                        <item xml:id="q30" n="30">
+                                            <dim unit="leaf">4</dim>
+                                            <locus from="vii" to="x"/> 
+                                        </item>
+                                    </list>
+                                </collation>
+                                <condition key="intact">
+                                    <locus target="#48 #79 #109v #163v #208 #210 #211"/> blank. 
```

Comment:
The information on blank leaves can be encoded as  `<measure unit="leaf" type="blank">7</measure><locus target="#48 #79 #109v #163v #208 #210 #211"/>` above in dimensions